llvm/lib/Passes/StandardInstrumentations.cpp

```diff
@@ -929,6 +942,11 @@ bool PrintIRInstrumentation::shouldPrintAfterPass(StringRef PassID) {
   return is_contained(printAfterPasses(), PassName);
 }
 
+bool PrintIRInstrumentation::shouldPrintAfterPassNumberCheck() {
```

Comment:
`shouldPrintAfterPassNumberCheck`/`shouldPrintAfterPassNumber` is a bit confusing. perhaps `shouldPrintAfter/BeforePassNumber` should be `shouldPrintAfter/BeforeSomePassNumber`, and `shouldPrintAfter/BeforePassNumberCheck` should be `shouldPrintAfter/BeforeCurrentPassNumber`?